cli.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import utils
+import master
+
+import typer
+
+from docker.errors import DockerException
+
+app = typer.Typer(add_completion=False)
+
+
+@app.command()
+def start_master(docker_base_url: str = 'unix://var/run/docker.sock', max_workers: int = 2):
```

Comment:
We are currently not using `max_workers` in this function - let's mark it as unused, just like you did in `master.py`

```suggestion
def start_master(docker_base_url: str = 'unix://var/run/docker.sock', _max_workers: int = 2):
```